<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotostationPlacerFeatureSelectionParametersControlWidget</class>
 <widget class="QWidget" name="PhotostationPlacerFeatureSelectionParametersControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhotostationPlacerFeatureSelectionParameters</string>
  </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_0">
       <property name="text">
         <string>inlierThreshold</string>
       </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="inlierThresholdSpinBox">
       <property name="toolTip">
         <string>Inlier threshold</string>
       </property>
       <property name="minimum">
        <double>-50000</double>
       </property>
       <property name="maximum">
        <double>50000</double>
       </property>
       <property name="singleStep">
        <double>1</double>
       </property>
       <property name="value">
        <double>5</double>
       </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
       <property name="text">
         <string>trackInlierThreshold</string>
       </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="trackInlierThresholdSpinBox">
       <property name="toolTip">
         <string>Track append threshold</string>
       </property>
       <property name="minimum">
        <double>-50000</double>
       </property>
       <property name="maximum">
        <double>50000</double>
       </property>
       <property name="singleStep">
        <double>1</double>
       </property>
       <property name="value">
        <double>3</double>
       </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
       <property name="text">
         <string>distanceLimit</string>
       </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="distanceLimitSpinBox">
       <property name="toolTip">
         <string>Track distance limit</string>
       </property>
       <property name="minimum">
        <double>-50000</double>
       </property>
       <property name="maximum">
        <double>50000</double>
       </property>
       <property name="singleStep">
        <double>1</double>
       </property>
       <property name="value">
        <double>1000</double>
       </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
       <property name="text">
         <string>detector</string>
       </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="detectorEdit">
       <property name="toolTip">
         <string>Detector</string>
       </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
       <property name="text">
         <string>descriptor</string>
       </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="descriptorEdit">
       <property name="toolTip">
         <string>Descriptor</string>
       </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
       <property name="text">
         <string>matcher</string>
       </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="matcherEdit">
       <property name="toolTip">
         <string>Matcher</string>
       </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
       <property name="text">
         <string>b2bThreshold</string>
       </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="b2bThresholdSpinBox">
       <property name="toolTip">
         <string>b2b threshold</string>
       </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
       <property name="text">
         <string>rmsePruningScaler</string>
       </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDoubleSpinBox" name="rmsePruningScalerSpinBox">
       <property name="toolTip">
         <string>RMSE pruning scaler</string>
       </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
       <property name="text">
         <string>maxPruningScaler</string>
       </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QDoubleSpinBox" name="maxPruningScalerSpinBox">
       <property name="toolTip">
         <string>Max pruning scaler</string>
       </property>
     </widget>
    </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <resources/>
 <connections>
  <connection>
   <sender>inlierThresholdSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PhotostationPlacerFeatureSelectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>trackInlierThresholdSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PhotostationPlacerFeatureSelectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>distanceLimitSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PhotostationPlacerFeatureSelectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>detectorEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PhotostationPlacerFeatureSelectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>descriptorEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PhotostationPlacerFeatureSelectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>matcherEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PhotostationPlacerFeatureSelectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>b2bThresholdSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PhotostationPlacerFeatureSelectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>rmsePruningScalerSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PhotostationPlacerFeatureSelectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>maxPruningScalerSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PhotostationPlacerFeatureSelectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
 <customwidgets>
 </customwidgets>
</ui>
